portal-db/api/scripts/set_authenticator_password.sql

```diff
@@ -0,0 +1,31 @@
+-- ============================================================================
```

Comment:
Rename the file to `local_development_only_set...sql` so there's never confusion